backend/globaleaks/handlers/authentication.py

```diff
@@ -52,7 +52,7 @@ def login_whistleblower(store, receipt, using_tor2web):
     """
     login_whistleblower returns the WhistleblowerTip.id
     """
-    hashed_receipt = security.hash_password(receipt, GLSettings.memory_copy.receipt_salt)
+    hashed_receipt = security.hash_password(receipt, GLSettings.memory_copy.password_salt)
```

Comment:
Why not have two different salts? Instead of reusing the basic auth salt for the receipt just keep the `receipt_salt`.